source_files/gpu.c

```diff
@@ -35,22 +42,20 @@
 #include "gpu.h"
 
 #define CUDA_ERROR_CHECK
-#define CudaSafeCall( err ) __cudaSafeCall( err, __FILE__, __LINE__ )
-#define CudaCheckError() __cudaCheckError( __FILE__, __LINE__ )
+#define cuda_safe_call( cuerr ) __cuda_safe_call( cuerr, __FILE__, __LINE__ )
```

Comment:
"Each name that contains a double underscore (_ _) or begins with an underscore followed by an uppercase letter (2.11) is reserved to the implementation for any use."
"All identifiers that begin with an underscore and either an uppercase letter or another underscore are always reserved for any use."
Please rename __cuda_safe_call to static inline cuda_safe_call

Reply:
you might also use a define instead of a static inline, but double underscore is a "meh"